src/dom/dom.js

```diff
@@ -1694,15 +1694,27 @@ p5.Element.prototype.html = function() {
  */
```

Comment:
add some explanation of the ability to add position type. right now the description indicates that it will always be position:absolute.
can you also add an example?